composer.json

```diff
@@ -13,6 +13,7 @@
         "symfony/messenger": "^4.4|^5.0",
         "symfony/polyfill-uuid": "^1.13",
         "symfony/routing": "^4.4|^5.0",
+        "symfony/security-bundle": "^5.0",
```

Comment:
I would make this optional, and then either not register the event listener if it's no available. OR, allow null to be passed into the constructor of the listener and if authorizationChecker is null, do nothing.

Reply:
Hmm, or does it make more sense to put the security logic directly into the controllers? It would be a bit simpler.

Reply:
It will indeed be simpler, but I prefer the listener solution, it would be made once for all, even if we add more controllers...